imports/ui/components/edit/edit-collection.jsx

```diff
@@ -106,4 +103,4 @@ let EditCollection = React.createClass({
   }
 })
 
-export default connect(state => state)(EditCollection)
```

Comment:
Ah, sorry.  So we're *not* exporting a connected component any more, since we don't need any of the Redux state or actions in this component.  So there's no point connecting it.